db/migrations/sqls/20200612085103-add-visit-status-up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE scheduled_calls_table ADD status varchar(20);
```

Comment:
How do we plan to deal with the existing visits in the scheduled_calls_table ? Should we update this migration to also update the status on any existing calls to be 'scheduled' (as all the completed visits will have been deleted)

Reply:
Backfill query added to migration.